components/server/ee/src/workspace/gitpod-server-impl.ts

```diff
@@ -2054,22 +2054,37 @@ export class GitpodServerEEImpl extends GitpodServerImpl {
         }
     }
 
+    async createOrUpdateStripeCustomerForTeam(ctx: TraceContext, teamId: string, currency: string): Promise<void> {
+        const user = this.checkAndBlockUser("createOrUpdateStripeCustomerForTeam");
+        const team = await this.guardTeamOperation(teamId, "update");
+        await this.ensureStripeApiIsAllowed({ team });
+        try {
+            let customer = await this.stripeService.findCustomerByTeamId(team!.id);
+            if (!customer) {
+                customer = await this.stripeService.createCustomerForTeam(user, team!);
+            }
+            await this.stripeService.setPreferredCurrencyForCustomer(customer, currency);
```

Comment:
Would this allow me to change my currency after I've got a subscription in one currency? And if so, is that a problem?

Reply:
> Would this allow me to change my currency after I've got a subscription in one currency?

Yes -- you can always cancel your Subscription, then buy a new Subscription in any available currency.

> And if so, is that a problem?

No -- this is a feature. If you're paying in USD, but want to change to EUR, you can.